spec.go

```diff
@@ -161,6 +163,13 @@ type ArtifactConfig struct {
 	Name string `yaml:"name,omitempty" json:"name,omitempty"`
 }
 
+// ArtifactConfigFileConfig is the configuration give for Artifact Config Files.
+// This is used to customize the settings for a config entry in the resulting artifact.
+type ArtifactConfigFileConfig struct {
+	ArtifactConfig
+	ReplaceOnUpdate bool `yaml:"replace,omitempty" json:"replace,omitempty"`
```

Comment:
Can you add a doc string for this?

Reply:
Actually reading up on `%config(noreplace)`, I think we should just assume this and use a normal `ArtifactConfig`.
If we need to tweak in the future we can add an option but I'm thinking `%config(noreplace)` is the normal correct behavior.

Reply:
sounds good to me, made this change.